.github/workflows/winget.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish to WinGet
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - autoupdate/stable-msi.json
+
+jobs:
+  publish:
+    # Action can only be run on windows
+    runs-on: windows-latest
+    steps:
+      - name: Should publish to WinGet?
+        run: |-
+          # Get the tag from the autoupdate/stable-msi.json
+          $githubRawUrl = "https://raw.githubusercontent.com/OpenKneeboard/OpenKneeboard/master/autoupdate/stable-msi.json"
+          $tag = (Invoke-RestMethod -Uri $githubRawUrl).tag_name
```

Comment:
TIL Invoke-RestMethod deals with JSON, nice.

Reply:
By the way, do you know PowerShell?

Reply:
Just what I've learned through github actions for windows/cross-platform stuff. Professionally, Linux/Mac exclusive, so still prefer POSIX shell for Linux/Mac-only stuff (I know powershell is available).